pkgs/top-level/all-packages.nix

```diff
@@ -10832,12 +10832,6 @@ in
 
   libcap = callPackage ../os-specific/linux/libcap { };
 
-  libcap_progs = libcap.bin;
```

Comment:
I'd personally prefer to move these to the `Deprecated aliases - for backward compatibility` (at the end of this file) instead of removing outright.